core-services/template-deprecation/_allowlist.yaml

```diff
@@ -3235,6 +3235,9 @@ templates:
         pull-ci-ComplianceAsCode-content-master-e2e-aws-rhcos4-moderate:
           generated: false
           kind: presubmit
+        pull-ci-open-cluster-management-observatorium-operator-release-2.1-test:
```

Comment:
That test doesn't exist, also you don't seem to use any template job as well.

Reply:
I have removed it.